examples/cpp_api_inference/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.5)
+project (CAI)
+
+set (MIGRAPHX_FOLDER AMDMIGraphX)
+set (MIGRAPHX_BUILD ${MIGRAPHX_FOLDER}/build)
```

Comment:
This should call `find_package(migraphx)`

Reply:
[Here](https://rocmdocs.amd.com/en/latest/Installation_Guide/Using-CMake-with-AMD-ROCm.html) is the guide for using cmake with rocm. We probably need to add the line `list (APPEND CMAKE_PREFIX_PATH /opt/rocm/hip /opt/rocm)` so it works with the examples with the installed migraphx packages.

Reply:
Fixed